style.css

```diff
@@ -0,0 +1,868 @@
+:root {
+    --text-color: white;
+    --items-border: 0.6rem;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+    list-style: none;
+}
+
+html {
+    scroll-behavior: smooth;
+}
+
+body {
+    background-color: rgb(36, 37, 130);
+}
+
+.flex-main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.grid-main {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 1.5rem;
+}
+
+.header {
+    background-color: #242582;
+    padding: 0.8rem 0;
+    justify-content: space-around;
+}
+
+.header a {
+    color: var(--text-color);
+    font-size: 1.5rem;
+    transition: all 0.5s;
+}
+
+.header a:hover {
+    transform: scale(1.3);
+    color: gold;
+}
+
+.header img {
+    border-radius: 50%;
+    width: 1.5rem;
+}
+
+.navbar {
+    background-color: #553d67;
+    padding: 0.5rem 1rem;
+    position: sticky;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    font-size: 1.3rem;
+    gap: 2.5rem;
+}
+
+.navbar > a {
+    flex-grow: 1;
+    text-align: center;
+}
+
+.nav-links {
+    flex-grow: 4;
+    justify-content: space-between;
+}
+
+.user-nav {
+    gap: 0.5rem;
+}
+
+.logo {
+    background-image: url(/src/images/winter-symbol.jpeg);
+    background-size: 100% 100%;
+    border-radius: var(--items-border);
+    padding: 0.4rem;
+}
+
+.logo span {
+    background-color: rgba(36, 37, 130, 0.5);
+    color: var(--text-color);
+    border-radius: var(--items-border);
+    font-weight: bolder;
+    padding: 0.2rem 0.3rem;
+}
+
+.navbar a, .navbar i {
+    color: var(--text-color);
+    border-radius: var(--items-border);
+    padding: 0.4rem;
+    transition: all 0.5s;
+}
+
+.navbar a:hover, .navbar i:hover {
+    background-color: #242582;
+    transform: scale(1.2);
+    padding: 0.2rem;
+}
+
+.landing {
+    max-width: 100%;
+    height: 85vh;
+    background-image: url(https://www.geico.com/living/wp-content/uploads/save-on-heating-bill.jpg);
+    background-size: 100% 100%;
+    position: relative;
```

Comment:
BG img is stetched, maybe better to set the size to cover.

Reply:
Actually I used cover at first, but the responsivity was less than to use 100% 100% (not even just one time 100%, or repeat(2, 100%), or using no repeat).
For most cases you're right, cover is well enough, but only for classes .landing and .cart I had to keep it as I used.